<launch>

<!-- 45  deg  0.785398 rad -->
<!-- 90  deg  1.570796 rad -->
<!-- 150 deg  2.61799 rad -->
<!-- 140 deg  2.44346 rad -->


<!-- ================= Compressor ======================= -->
    <node pkg="wp3_compressor" type="wp3_compressor_node" name="jetson6_comp_velodyne">
        <param name="resolution" value="0.04" />
        <param name="input_topic" value="/velodyne_points" />
        <param name="input_type" value="1" />
        <param name="output_topic" value="/roof/velodyne_comp" />
        <param name="local_frame" value="velodyne" />
        <param name="global_frame" value="myworld" />
        <param name="min_x" value="0.0" />
        <param name="max_x" value="10.0" />
        <param name="min_y" value="0.0" />
        <param name="max_y" value="10.0" />
        <param name="min_z" value="0.0" />
        <param name="max_z" value="5.0" />
        <param name="crop"  value="true" />
    </node>   

</launch>

